components/color-picker/index.en-US.md

```diff
@@ -46,12 +46,12 @@ Common props ref：[Common props](/docs/react/common-props)
 | arrow | Configuration for popup arrow | `boolean \| { pointAtCenter: boolean }` | true | |
 | children | Trigger of ColorPicker | React.ReactNode | - | |
 | defaultValue | Default value of color | string \| `Color` | - | |
-| defaultFormat | Default format of color | `rgb` \| `hex` \| `hsb` | - | 5.9.0 |
+| defaultFormat | Default format of color | `rgb` \| `hex` \| `hsb` | `hex` | 5.9.0 |
```

Comment:
It doesn't have a default value. but from [here](https://github.com/ant-design/ant-design/blob/668fa4a7/components/color-picker/components/ColorInput.tsx#L51-L53), it can be found that actually, all unsupported data is converted to hex.